lib/manageiq/loggers/journald.rb

```diff
@@ -0,0 +1,54 @@
+require "systemd-journal"
```

Comment:
Can you move this to "inside" the initialize method?  That way we can make it an option gem requirement.

Reply:
:+1: